src/basic/virt.h

```diff
@@ -32,6 +32,7 @@ enum {
         VIRTUALIZATION_OPENVZ,
```

Comment:
So we will now have a situation where "microsoft" is one thing, and "wsl" is another. That's rather confusing. The problem was introduced when we decided to use "microsoft" as the name of a virtualization mechanism. We should probably rename it to something more specific. But that's independent of this PR.

Reply:
Done in a separate commit.